bundles/org.openhab.ui/web/src/components/widgets/system/oh-video.vue

```diff
@@ -6,7 +6,8 @@
       :stunServer="config.stunServer"
       :candidatesTimeout="config.candidatesTimeout"
       :startManually="config.startManually"
-      :hideControls="config.hideControls" />
+      :hideControls="config.hideControls"
+      :posterURL="config.posterURL" />
```

Comment:
Wouldn't you need to add this to the `<oh-video-videojs>` alternative below too?

Reply:
🤦   Good catch, i somehow lost that in a refactor.  Fixed.